INFO:root:bot has started
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "D:\”л√“”\олимпиады\хакатон\tinder\venv\lib\site-packages\telegram\ext\dispatcher.py", line 555, in process_update
    handler.handle_update(update, self, check, context)
  File "D:\”л√“”\олимпиады\хакатон\tinder\venv\lib\site-packages\telegram\ext\conversationhandler.py", line 626, in handle_update
    new_state = handler.handle_update(update, dispatcher, check_result, context)
  File "D:\”л√“”\олимпиады\хакатон\tinder\venv\lib\site-packages\telegram\ext\handler.py", line 198, in handle_update
    return self.callback(update, context)
  File "D:\”л√“”\олимпиады\хакатон\tinder\form.py", line 49, in form_subjects_know
    contex.user_data["form"]["bad_know"][-1][1] = int(user_message)
TypeError: 'int' object does not support item assignment
